docs/content/preview/explore/query-1-performance/pg-stat-progress-copy.md

```diff
@@ -0,0 +1,174 @@
+
+    ```sql
+    create table test_copy ( h1 int, r1 int, v1 int, primary key (h1, r1));
+    ```
+
+### Run a successful COPY command
+
+1. Copy `test_data.csv` using the following copy command:
+
+    ```sql
+    \copy test_copy from 'test_data.csv'  (DELIMITER ',');
+    ```
+
+    ```output
+    COPY 100000
+    ```
+
+1. Verify `yb_status` using the view.
+
+    ```sql
+    select * from pg_stat_progress_copy ;
+    ```
+
+    You should see output similar to the following:
+
+    ```output
+     pid  | datid | datname  | relid |  command  | type | yb_status | bytes_processed | bytes_total | tuples_processed | tuples_excluded
+    -------+-------+----------+-------+-----------+------+-----------+-----------------+-------------+------------------+-----------------
+    74390 | 13288 | yugabyte | 16394 | COPY FROM | PIPE | SUCCESS   |         1966685 |           0 |           100000 |               0
+    (1 row)
+    ```
+
+### Verify a COPY command status in progress
+
+Copy `test_data.csv` using the following copy command and check the status in parallel from another terminal.
+
+```sql
+\copy test_copy from 'test_data.csv'  (DELIMITER ',');
+```
+
+```sql
+select * from pg_stat_progress_copy ;
+```
+
+You should see output similar to the following:
+
+```output
+ pid  | datid | datname  | relid |  command  | type |  yb_status  | bytes_processed | bytes_total | tuples_processed | tuples_excluded
+-------+-------+----------+-------+-----------+------+-------------+-----------------+-------------+------------------+-----------------
+74390 | 13288 | yugabyte | 16404 | COPY FROM | PIPE | IN PROGRESS |          766682 |           0 |            40000 |               0
+(1 row)
+```
+
+### Verify an interrupted COPY operation
+
+1. Copy `test_data.csv` and interrupt it using the following commands:
+
+    ```sh
+    \copy test_copy from 'test_data.csv'  (DELIMITER ',');
+    ```
+
+    ```output
+    ^CCancel request sent
+    ERROR:  canceling statement due to user request
+    CONTEXT:  COPY test_copy, line 55184: "55184, 55184, 55184"
+    ````
+
+1. Verify `yb_status` using the view.
+
+     ```sql
+    select * from pg_stat_progress_copy ;
+    ```
+
+    ```output
+      pid  | datid | datname  | relid |  command  | type | yb_status | bytes_processed | bytes_total | tuples_processed | tuples_excluded
+    -------+-------+----------+-------+-----------+------+-----------+-----------------+-------------+------------------+-----------------
+     74390 | 13288 | yugabyte | 16399 | COPY FROM | PIPE | ERROR     |          766682 |           0 |            40000 |               0
+    (1 row)
+    ```
+
+    The default for `ROWS_PER_TRANSACTION` is 20000 and the interruption occurred on row 55184 in the example. So the last 15184 rows were not persisted and the first 40000 rows were persisted.
```

Comment:
May be update this one as following --- 

The default for `ROWS_PER_TRANSACTION` is 2000 and the interruption occurred on row 55184 in the example.  As a result of this  first two transaction  of 20K rows were persisted resulting in 40K rows reported by tuples_processed. The next  in-progress transaction  was interrupted and due to this reason remaining 15184 rows were not persisted .

Reply:
default for ROWS_PER_TRANSACTION is 20k . I noted wrong in comment above as 2k.